app/views/ideas/show.html.erb

```diff
@@ -1,5 +1,5 @@
-<% @user.ideas.each do |idea| %>
+<% current_user.ideas.each do |idea| %>
 <%= idea.name %><br>
```

Comment:
here is where i added the logic for current_user instead of @user to be more specific but is that really important? the controller seems to know who @user is so maybe this is my issue? I jsut want to be more specific and explicit

Reply:
and here I define current_user but this doesn't even get hit       👎

Reply:
Does your controller have @user in it and is it equal to current user?